src/tests/topp/ProteomicsLFQ_2_out.mzTab

```diff
@@ -65,146 +65,146 @@ PRH	accession	description	taxid	species	database	database_version	search_engine
 PRT	tr|A9GID7|A9GID7_SORC5	Putative uncharacterized protein OS=Sorangium cellulosum (strain So ce56) GN=sce3093 PE=4 SV=1	null	null	18Protein_SoCe_Tr_detergents_trace_target_decoy	null	null	0.25	null	null	0.044052863436123	null	null	null	null	null	null	null	null	null	null	null	null	null	null	2	0	protein_details
 PRT	P00761|TRYP_PIG	Trypsin - Sus scrofa (Pig).	null	null	18Protein_SoCe_Tr_detergents_trace_target_decoy	null	null	0.25	null	null	0.077922077922078	null	null	null	null	null	null	null	null	null	null	null	null	null	null	2	0	protein_details
 PRT	P02769|ALBU_BOVIN	Serum albumin - Bos taurus (Bovine).	null	null	18Protein_SoCe_Tr_detergents_trace_target_decoy	null	null	0.25	null	null	0.401976935749588	null	null	null	null	null	null	null	null	null	null	null	null	null	null	36	0	protein_details
-PRT	P00761|TRYP_PIG	Trypsin - Sus scrofa (Pig).	null	null	18Protein_SoCe_Tr_detergents_trace_target_decoy	null	null	0.25	P00761|TRYP_PIG	null	0.077922077922078	0.0	7.7557855e06	0.0	0.0	null	null	7.7557855e06	null	null	0.0	null	null	null	null	null	null	single_protein
-PRT	P02769|ALBU_BOVIN	Serum albumin - Bos taurus (Bovine).	null	null	18Protein_SoCe_Tr_detergents_trace_target_decoy	null	null	0.25	P02769|ALBU_BOVIN	590-UNIMOD:4	0.401976935749588	1.34777856e08	1.85369936e08	1.47976016e08	1.34777856e08	null	null	1.85369936e08	null	null	1.47976016e08	null	null	null	null	null	null	single_protein
+PRT	P00761|TRYP_PIG	Trypsin - Sus scrofa (Pig).	null	null	18Protein_SoCe_Tr_detergents_trace_target_decoy	null	null	0.25	P00761|TRYP_PIG	null	0.077922077922078	0.0	1.0431346e07	0.0	0.0	null	null	1.0431346e07	null	null	0.0	null	null	null	null	null	null	single_protein
```

Comment:
why are these changes so high?

Reply:
I think already little shifts in the fitted apex can yield vastly different integrated areas.